src/quote/QuoteSchema.ts

```diff
@@ -0,0 +1,68 @@
+const QuoteSchema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  $id: 'KILT:quote:QUOTEHASH',
+  type: 'object',
+  title: 'Quote',
+  version: '1.0.0',
+  properties: {
+    attesterAddress: {
+      type: 'string',
+      title: 'The Attester ID Schema',
+    },
+    cTypeHash: {
+      type: 'string',
+      title: 'The Ctypehash Schema',
+    },
+    cost: {
+      type: 'object',
+      title: 'The Price Schema',
```

Comment:
Question/Suggestion:
What are `title` fields used for? Why is `Schema` appended to each?

Reply:
`Title` is unnecessary and I removed it.